src/lib/structures/game/BinbGame.ts

```diff
@@ -151,19 +165,43 @@ export class BinbGame extends Game {
 	}
 
 	protected async getPlayers(voiceChannelMembers: Collection<string, GuildMember>) {
-		const joinButton = new MessageActionRow() //
+		const row = new MessageActionRow() //
 			.addComponents(
 				new MessageButton() //
-					.setCustomId(CustomIds.Join)
+					.setCustomId(CustomIds.Join + voiceChannelMembers.first()!.guild.id)
 					.setLabel('Join Game')
 					.setStyle(Constants.MessageButtonStyles.SUCCESS)
 			);
 
-		const embed = createEmbed(`Click the button to join this round of ${this.gameType} music trivia!`, EmbedColor.Primary);
-		await Promise.all(voiceChannelMembers.map((member) => member.send({ embeds: [embed], components: [joinButton] })));
+		const embed = createEmbed(`Click the button to join this game of ${this.gametype} music trivia!`);
+		const messagePromises = voiceChannelMembers.map((member) => member.send({ embeds: [embed], components: [row] }).catch(() => null));
+		const messages = await Promise.all(messagePromises);
```

Comment:
It's also an idea to use Promise.allSettled and filter via status and not have to catch all of them in the map but idk there's no problem with this currently